apps/api/src/common/util.spec.ts

```diff
@@ -26,21 +26,41 @@ describe('util', () => {
     expect(decrypted).toEqual(plaintext)
   })
 
+  it('should fail to encrypt and decrypt a string', () => {
+    const keyPair = createKeyPair()
+    const plainText = 'hello world'
+    const encrypted = encrypt(keyPair.publicKey, plainText)
+    const modifiedText = 'modified hello world'
+    const decrypted = decrypt(keyPair.privateKey, encrypted)
+    expect(decrypted).not.toEqual(modifiedText)
+  })
+
+  const object = {
```

Comment:
Remove this block aswell

Reply:
Okay, then I'll move this block  to the `should exclude fields` test as it was earlier

Reply:
sure!